File: java/program/Product.java
//Kelas Induk
class Product{
    //private atributes.
    private int idProduct;
    private String name;
    private String brand;
    private String price;

    /* Constructors */

    public Product(){
        this.idProduct = 0;
        this.name = "";
        this.brand = "-";
        this.price = "-";
    }

    public Product(int idProduct, String name, String brand, String price){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    /* Getter and Setter. */

    //get id product
    public int getIdProduct(){
        return this.idProduct;
    }

    //set id product
    public void setIdProduct(int idProduct){
        this.idProduct = idProduct;
    }

    //get name
    public String getName(){
        return this.name;
    }

    //set name
    public void setName(String name){
        this.name = name;
    }

    //get brand
    public String getBrand(){
        return this.brand;
    }
    //set brand
    public void setBrand(String brand){
        this.brand = brand;
    }

    //get price
    public String getPrice(){
        return this.price;
    }

    //set price
    public void setPrice(String price){
        this.price = price;
    }
}

File: php/Product.php
<?php

class Product{
    private $idProduct = 0;
    private $name = "";
    private $brand = "";
    private $price = "";

    public function __construct($idProduct = 0, $name = "", $brand = "", $price = ""){
        $this->idProduct = $idProduct;
        $this->name = $name;
        $this->brand = $brand;
        $this->price = $price;
    }

    public function setIdProduct($idProduct){
        $this->idProduct = $idProduct;
    }

    public function getIdProduct(){
        return $this->idProduct;
    }

    public function setName($name){
        $this->name = $name;
    }

    public function getName(){
        return $this->name;
    }

    public function setbrand($brand){
        $this->brand = $brand;
    }

    public function getbrand(){
        return $this->brand;
    }

    public function setprice($price){
        $this->price = $price;
    }

    public function getprice(){
        return $this->price;
    }
}

?>